EdFi.Ods.Utilities.Migration/Scripts/02Upgrade/v50_to_v51/01 Structure/1040-CreateFunctionChangeGetMaxChangeVersionForV51.sql

```diff
@@ -0,0 +1,25 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (
```

Comment:
These is related to the change query feature. if you compare the database reference scripts in EdFi.Ods.Standard\Artifacts\MsSql\Structure\Ods\Changes for v5.0.0 and v.5.1.0 and don't see any differences we should remove this file.

Reply:
done. thanks